src/nbl/video/CVulkanPhysicalDevice.h

```diff
@@ -99,6 +100,24 @@ class CVulkanPhysicalDevice final : public IPhysicalDevice
                 m_limits.maxRayHitAttributeSize = rayTracingPipelineProperties.maxRayHitAttributeSize;
             }
         }
+
+        asset::IGLSLCompiler::E_SPIRV_VERSION spirvVersion;
+        uint32_t minorVersion = VK_API_VERSION_MINOR(m_limits.apiVersion);
+        switch (minorVersion)
+        {
+        case 0:
+            spirvVersion = asset::IGLSLCompiler::ESV_1_0; break;
+        case 1:
+            spirvVersion = asset::IGLSLCompiler::ESV_1_3; break;
+        case 2:
+            spirvVersion = asset::IGLSLCompiler::ESV_1_5; break;
+        default:
+            assert(!"Invalid Vulkan minor version!");
```

Comment:
if you dont want the program to terminate, dont use an `assert`

Reply:
Resolved